### DISCUSSION TITLE
any recommendation for auth?

### DISCUSSION THREAD

**Discussion by rbartholomay** [quasarframework/quasar#14651] [General - Components / Directives / etc]
Hi there,

is there any recommendation for using auth in quasar with typescript, pinia?

Thanks!

René

**Comment by MilosPaunovic** [comment]
Take a look at Luke's https://quasar.vueauth.com/

**Comment by rbartholomay** [comment]
Do you know if this is already production ready?  This looks very good... In my case I need an auth provider with jwt against an api. So I hopefully find out how to add a simple provider...

Thanks...

**Reply by MilosPaunovic** [reply]
As Luke and the docs say `Done. In less than three minutes you're good to go.`

https://quasar.vueauth.com/guide/introduction.html#lightning-fast-setup
